js/script.js

```diff
@@ -75,7 +97,6 @@ function changeTrack(sourceUrl) {
 //Animating the slide show
 
 var slideIndex = 1;
-showSlides(slideIndex);
```

Comment:
since we're hiding `#locationPopup` to start and the `.locations` click handler is already calling `currentSlide(1)`, we don't need this anymore.